erts/emulator/ryu/update.sh

```diff
@@ -76,24 +79,26 @@ incorporate them into to_chars.h.
 
 Once done, update xcharconv_ryu.h.sha with the new sha. i.e.
 
-echo "${STL_SHA}" > ${ERL_TOP}/erts/emulator/ryu/xcharconv_ryu.h.sha
+echo "${STL_CHARCONV_SHA}" > ${ERL_TOP}/erts/emulator/ryu/xcharconv_ryu.h.sha
```

Comment:
if the `update-vendored-deps.sh` is going to run the updates, then some script should do this update for the `${ERL_TOP}/erts/emulator/ryu/xcharconv_ryu.h.sha`, isn't it?

Reply:
I've updated it so that a comment is posted to the PR that the update needs to be done manually in this scenario.